docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3.7"
+services:
+  todo-service-database:
+    image: mongo:latest
```

Comment:
`latest` tag is a bad practice, use concrete version of image

Reply:
latest → 5.0